### ISSUE TITLE
Missed readings triggers too fast or needlessly

### ISSUE THREAD

**Issue by LiroyvH** [Faifly/xDrip#347]
The missed readings alarm seems to trigger after only about 2 to 3 minutes worth of "missed readings"; so in reality there aren't really any missed readings. Suggesting setting this to a sane value (at least 10 minutes) or allow changing the trigger value.

-edit- Its strange though, the timer is set to 5.5 * 60 seconds which ought to be 5m30s before it triggers. I wonder if it simply always triggers after this time period. Set it to 10.5 and re-built, will see what happens. 

-edit2- Nope this just makes it trigger later, but it will still keep triggering; now even with last reading being 1 minute ago (or does it alert on each new reading for some reason? Not sure, but there's definitely something wrong with it heh.)

-edit3- Since I had previously used "Download nightscout data", which caused multiple problems, perhaps its also triggering these alarms based on downloaded nightscout data. 

-edit4-  Completely wiped the app, re-built it, setup Nightscout again but this time didn't have it download any historic data. No alarms are being triggered needlessly now by the looks of it. (Fingers crossed.) 

-edit5- Nope still triggers randomly. 04:34 readings come in. 04:36: missed readings triggers. Alas, I'm out of ideas.

**Comment by NSFriend** [comment]
@LiroyvH Have you paired app with transmitter, started sensor and used app in the master mode?

**Comment by LiroyvH** [comment]
@NSFriend Correct, I was using the app in master mode and was getting readings. Despite getting readings, it still kept triggering the missed readings alarm.

**Comment by NSFriend** [comment]
@LiroyvH Please wipe the app from your device and reinstall from latest commit adae8bf
here here https://github.com/Faifly/xDrip/tree/develop

**Comment by LiroyvH** [comment]
> @LiroyvH Please wipe the app from your device and reinstall from latest commit [adae8bf](https://github.com/Faifly/xDrip/commit/adae8bf8e46a5cd47badf9e8b321125b313579c6)
> here here https://github.com/Faifly/xDrip/tree/develop

Hello,

Also re: #346, I've installed the updated build as requested.
I no longer seem to be getting any missed readings alarms whilst getting readings and it looks like the app also did not download a huge amount of nightscout treatments this time around.

Both issues, at first glance, appear to be resolved - thanks!
